includes/Order/functions.php

```diff
@@ -885,6 +886,9 @@ function dokan_order_csv_export( $orders, $file = null ) {
                 case 'order_total':
                     $line[ $row_key ] = $the_order->get_total();
                     break;
+                case 'earnings':
+                    $line[ $row_key ] = dokan()->commission->get_earning_by_order($the_order);
```

Comment:
add a single space before and after $the_order

Reply:
Bhai, please check now